src/components/Book.js

```diff
@@ -1,20 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/books';
 
-export default function User(props) {
+export default function Book(props) {
```

Comment:
- [ ] kindly use ES6 syntax. and make sure that you changed all you used the `function` to const as well as the arrow function.